tests/install/test_builder.py

```diff
@@ -111,6 +123,27 @@ def test_rai_builder_raises_if_attempting_to_place_deps_in_nonempty_dir(
         rai_builder._fetch_deps_for(device)
 
 
+invalid_build_arm64 = [
+    dict(build_tf=True, build_onnx=True),
+    dict(build_tf=False, build_onnx=True),
+    dict(build_tf=True, build_onnx=False),
+]
+invalid_build_ids = [
+    ",".join([f"{key}={value}" for key, value in d.items()])
+    for d in invalid_build_arm64
+]
+
+
+@pytest.mark.parametrize("build_options", invalid_build_arm64, ids=invalid_build_ids)
+def test_rai_builder_raises_if_unsupported_deps_on_arm64(monkeypatch, build_options):
```

Comment:
ditto unneeded `monkeypatch`
```suggestion
def test_rai_builder_raises_if_unsupported_deps_on_arm64(build_options):
```